helpers/search_helper.py

```diff
@@ -55,6 +56,31 @@ def update_team_location_index(cls, team):
     def remove_team_location_index(cls, team):
         search.Index(name=cls.TEAM_LOCATION_INDEX).delete(team.key.id())
 
+    @classmethod
+    def update_match_number_index(cls, match):
```

Comment:
A task that enqueues a call to this should be added to `MatchManipulator.postUpdateHook()` to keep the search index in sync with this. Add to the queue `search-index-update`.